lib/screens/chat_screens/chat_sceen.dart

```diff
@@ -0,0 +1,255 @@
+import 'package:date_time_picker/date_time_picker.dart';
+import 'package:flutter/material.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:project_eureka_flutter/components/eureka_appbar.dart';
+import 'package:project_eureka_flutter/services/email_auth.dart';
+
+final _firestore = FirebaseFirestore.instance;
+User loggedInUser = EmailAuth().getCurrentUser();
+
+class ChatScreen extends StatefulWidget {
+  final String fromId;
+  final String recipient;
+  ChatScreen(this.fromId, this.recipient);
+
+  @override
+  _ChatScreenState createState() => new _ChatScreenState(fromId, recipient);
+}
+
+class _ChatScreenState extends State<ChatScreen> {
+  final messageTextController = TextEditingController();
+  final _auth = FirebaseAuth.instance;
+  String fromId;
+  String recipient;
+  _ChatScreenState(this.fromId, this.recipient);
+
+  String messageText;
+  String groupChatId;
+  String userId;
+
+  @override
+  void initState() {
+    super.initState();
+    getCurrentuser();
+    userId = loggedInUser.uid;
```

Comment:
these two lines do the same thing, since both is getting the id from `FirebaseAuth.instance`